src/dbos.ts

```diff
@@ -524,10 +512,11 @@ export class DBOS {
   // Globals
   //////
   static #dbosConfig?: DBOSConfig;
-  static #runtimeConfig?: DBOSRuntimeConfig = undefined;
+  static #poolConfig?: PoolConfig;
+  static #port?: number;
 
-  static get dbosConfig(): DBOSConfig | undefined {
-    return DBOS.#dbosConfig;
+  static get dbosConfig(): ReadonlyArray<DBOSConfig & { poolConfig?: PoolConfig }> {
```

Comment:
I'd remove this and not expose the DBOS config publicly at all.

Reply:
we have tests that use it. Will clean this up in a later PR

Reply:
FYI, the wonky addition of poolConfig to this property is for test usage